src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/TwigBundle/views/Exception/error.html.twig

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+<div id="page" class="container">
+
+  <!-- Content -->
+  <section id="content">
+  	<div class="row">
+  		<article class="news_item error_page">
+  			<div class="content content_text ">
+				<h2>Oops... We have encountered an error.</h2>
+				<p>Please try by following links in main navigation.</p>
+			</div>
+		</article>
+	</div>
+  </section><!-- End Content -->
+
+</div>
+{% endblock %}
+
+{% block template_code %}
+<pre class="js-template-code" style="display:none;">{{ source(_self.getTemplateName())|escape('html') }}</pre>
+{% endblock %}
+
+
```

Comment:
remove one new line which is not needed, only one new line should be added at the end of the file